scripts/beck/st/README

```diff
@@ -0,0 +1,65 @@
+The script will create hole in metadata as per user choice. There are n number of ways to create hole using error_injection.py script.
```

Comment:
Please explain, "create hole", "error injection".
What is the difference?
Why do you say "create hole?"
Do you mean to insert a segment with all zeroes?
"error injection"? What error is going to injected into the metadata file?
Just some array of random bytes with random size?
Or some array of all zeroes with random size?
At any offset in the metadata file?
or it will overwrite some records with random bytes? At the record boundary?

Please explain all these topics.

Reply:
As per Motr it is error injection in Metadata means corrupt some entries with over-writing some known pattern data on a particular location
As per S3 they call it a Hole. So kept both for better understanding 

Do you mean to insert a segment with all zeroes?
No, Not segment. Segment is different
"error injection"? What error is going to inject into the metadata file?
Write some unexpected data is Error in metadata
Just some array of random bytes with random size?
No, not that straight forward. Fix pattern "111122223334444" pattern
Or some array of all zeroes with random size?
At any offset in the metadata file?
Depend on what you are testing.
or it will overwrite some records with random bytes? At the record boundary?
Now it will overwrite some record depending on what functionality you are choosing at a particular offset of a record.